File: pipe_me.r
#!/usr/bin/env Rscript
n <- as.integer(commandArgs(trailingOnly = TRUE)) # Leemos un entero como argumento (opcional)

f <- file("stdin")

open(f)

while(length(line <- readLines(f, n = 1)) > 0) {
   # Aquí habría que hacer cosas más interesantes con la línea
   print(paste0("Linea desde R: " , line))
}

close(f)

File: pipe_me.py
#!/usr/bin/env python
# coding: utf-8

import re
import sys

# n = int(sys.argv[1]) # Leemos un entero como argumento (opcional)

while True:
  linea = sys.stdin.readline()

  if not linea:
    break
  # Hacemos algo con la línea
  print("Linea desde python: {}".format(linea))
